theano/tensor/nnet/tests/test_conv3d2d.py

```diff
@@ -76,10 +76,9 @@ def pyconv3d(signals, filters):
 
 
 def check_diagonal_subtensor_view_traces(fn):
-    def check_node_type(node):
-        return isinstance(node.op, (DiagonalSubtensor, IncDiagonalSubtensor))
-    assert check_stack_trace(fn, ops_to_check=check_node_type,
-                             bug_print='ignore')
+    assert check_stack_trace(
+        fn, ops_to_check=(DiagonalSubtensor, IncDiagonalSubtensor),
+        bug_print='ignore')
```

Comment:
Why 'ignore'? Is there a case where they are not there? Can you add a comment about it?